sqlstore/src/main/resources/db/migration/V8010__add_sop_fields.sql

```diff
@@ -0,0 +1,13 @@
+
+CREATE TABLE SopField (
+    sopFieldId BIGINT AUTO_INCREMENT PRIMARY KEY,
+    sopId BIGINT NOT NULL,
+    name VARCHAR(255) NOT NULL,
+    units VARCHAR(50),
+    fieldType VARCHAR(50) NOT NULL DEFAULT 'TEXT',
```

Comment:
Don't set a default at the database level as this doesn't really help with anything